packages/config-loader/src/loader.ts

```diff
@@ -107,6 +107,7 @@ export async function loadConfig(
     remote: options.remote && {
       reloadInterval: { seconds: options.remote.reloadIntervalSeconds },
     },
+    watch: Boolean(options.watch),
```

Comment:
We'll want this to default to true, I think it's best to pass through as is (without `Boolean(...)`)

Reply:
If I pass it through, I must convert it into a boolean value after.

Reply:
By making it a boolean here we will default it to `false`, which we don't want. It needs to be passed through as `undefined` if it is